cloud/releases/cloud-release-notes.md

```diff
@@ -3,6 +3,119 @@ id: cloud-release-notes
+- [Metrics Graph widget] Added curve type format setting.
+- [Metrics Graph widget] Added multiple options to customize graph.
+- [Metrics Graph widget] Enabled visualization using bar chart and stacked bar chart.
+- [Metrics Graph widget] Distinguished curves for same metric name, by automatically adjusting colors.
+- [Resource Table widget] Moved display type selection into the widget.
+
+</details>
+
+<details>
+  <summary>Bug fixes</summary>
+
+- [Dataset] Removed incompatible resources when filtering using multiple resource types.
+- Whitelist errors now have a ERROR level in Gorgone logs.
+
+</details>
+
+## Centreon gorgone
+
+### 24.07.0
+
+Release date: `August 06 , 2024`
+
+<details>
+  <summary>Bug fixes</summary>
+
+- [MBI] Fixed the perfdataStatisticsBuilder.pl script: when using the '-r --centile-only' parameters, the mod_bi_metrichourlyvalue table is no longer truncated.
+- Whitelist errors now have a ERROR level in Gorgone logs.
+
+</details>
+
+## Centreon collect
+
+### 24.07.0
+
+Release date: `August 06 , 2024`
+
+<details open>
+  <summary>Enhancements</summary>
+
+- [Engine] The customvariables configuration is now independent from the main Engine code.
+- [Engine] The whitelist now works with all the commands executed by Engine, including notifications and handlers.
+- [Engine] Engine can now handle extra configuration files to complete/overload the centengine.cfg file. This enhancement allows to keep a custom configuration unchanged by the configuration export.
+- [Engine] Added a host_down_disable_service_checks option.
+- The spdlog logger library has been mutualized for Engine and Broker, as well as the log files. This change will not impact Broker's logs, but Engine will log all its messages (including broker-module's messages) in centengine.log.
+- Whitelist errors now have a ERROR level in Gorgone logs.
+
+</details>
+
+## Centreon map
+
+### 24.07.0
+
+Release date: `August 06 , 2024`
+
+<details open>
+  <summary>Enhancements</summary>
+
+- [Editor] Added option to not display label for Containers.
+- Display a message on the MAP widget when displayed in a public page.
+- [GeoViews] Redesigned the edit modal.
+- [VIEWER] Added possibility to display status as bg color for resource in weather style.
+
+</details>
+
+<details>
+  <summary>Bug fixes</summary>
+
+- [Map list] Fixed an issue that could prevent the map list from being displayed.
+- Status of BA is not aligned with the status in Resource Status.
+
+</details>
+
+## Centreon mbi
+
+### 24.07.0
+
+Release date: `August 06 , 2024`
+
+<details>
+  <summary>Bug fixes</summary>
+
+- - Fixed the perfdataStatisticsBuilder.pl script: when using the '-r --centile-only' parameters, the mod_bi_metrichourlyvalue table is no longer truncated.
+
+</details>
+
+## Centreon BAM
+
+### 24.07.0
+
+Release date: `August 06 , 2024`
+
+<details>
+  <summary>Bug fixes</summary>
+
+- [BAM] Fixed Deprecation due to Symfony Upgrade.
+
+</details>
+
+
```

Comment:
## August xx, 2024

### Open Access to Centreon APIs

It is now possible to interact with Centreon through APIs and therefore integrate Centreon with any monitoring or data processing tool you may use, automatize monitoring configuration updates and any other use case you may have in mind. Access to APIs is granted thanks to tokens linked to contacts for access right management, but completely independent in terms of lifecycle.
API documentation is available here (link)

### Centreon Dashboards

It is now possible to declare dashboards playlists as public. An URL is genrated and give access to the dashboard playlist without authentication. This feature is ideal to display dahboards on big screens.
Note that this feature was implemented in a completely secured way. No confidential information is carried by the URL, nor it is possible to have access to any other information or data than the one displayed.
Note: MAP widget is not supported yet on public playlists.

This version of Centreon Dashboards comes with many other improvements. 
Graph widget allows much more configuration (automatic curves color adjustement for metric of same kind, bar chart and stacked bar chart presentation, 3 different curves design, and many others).
Widgets are also now more compact, allowing to display more of them on a single display.
And finally, freshness of data is now exposed and user can reload data at the widget level.